contrib/runners/action_chain_runner/action_chain_runner.py

```diff
@@ -278,104 +278,210 @@ def pre_run(self):
         try:
             self.chain_holder.validate()
         except Exception as e:
-            raise runnerexceptions.ActionRunnerPreRunError(e.message)
+            raise runner_exc.ActionRunnerPreRunError(e.message)
 
     def run(self, action_parameters):
-        # holds final result we store.
+        # Run the action chain.
+        return self._run_chain(action_parameters)
+
+    def pause(self):
+        # Identify the list of action executions that are workflows and cascade pause.
+        for child_exec_id in self.execution.children:
+            child_exec = ActionExecution.get(id=child_exec_id, raise_exception=True)
+            if (child_exec.runner['name'] in action_constants.WORKFLOW_RUNNER_TYPES and
```

Comment:
Shouldn't we just check for action-chain workflows here?

Or do we also want to pause a child task in case it's an action which is a Mistral workflow?